JWBestPracticeApps/Podfile

```diff
@@ -4,7 +4,7 @@
 # use_frameworks!
 
 def common_JW
-    pod 'JWPlayer-SDK', '~> 3.11.0'
+    pod 'JWPlayer-SDK', '~> 3.12.0'
```

Comment:
@s-seibert you don't need to change this because '~>' means it's 3.11 and up, so when we released 3.12.0, it would have grabbed 3.12.0, and any newer version going forward.